src/main/java/com/avispl/symphony/dal/communicator/lg/lcd/LgLCDDevice.java

```diff
@@ -114,55 +186,330 @@ public void controlProperties(List<ControllableProperty> controllableProperties)
+				String group = property + LgLCDConstants.HASH;
+				if (property.contains(LgLCDConstants.HASH)) {
+					propertyKey = propertyList[1];
+					group = propertyList[0] + LgLCDConstants.HASH;
+				} else {
+					propertyKey = property;
+				}
+				LgControllingCommand lgControllingCommand = LgControllingCommand.getCommandByName(propertyKey);
+				switch (lgControllingCommand) {
+					case VOLUME:
+						String dataConvert = Integer.toHexString((int) Float.parseFloat(value));
+						sendRequestToControlValue(commandNames.VOLUME, dataConvert.getBytes(StandardCharsets.UTF_8));
+						stats.put(group + LgLCDConstants.VOLUME_VALUE, String.valueOf((int) Float.parseFloat(value)));
+						break;
+					case MUTE:
+						String mute = LgLCDConstants.UNMUTE_VALUE;
+						if (String.valueOf(LgLCDConstants.NUMBER_ONE).equals(value)) {
+							mute = LgLCDConstants.MUTE_VALUE;
+						}
+						sendRequestToControlValue(commandNames.MUTE, mute.getBytes(StandardCharsets.UTF_8));
+						break;
+					case BACKLIGHT:
+						dataConvert = Integer.toHexString((int) Float.parseFloat(value));
+						sendRequestToControlValue(commandNames.BACKLIGHT, dataConvert.getBytes(StandardCharsets.UTF_8));
+						stats.put(group + LgLCDConstants.BACKLIGHT_VALUE, String.valueOf((int) Float.parseFloat(value)));
+						break;
+					case INPUT_SOURCE:
+						dataConvert = InputSourceDropdown.getValueOfEnumByNameAndType(value, isInputTypeDTV);
+						sendRequestToControlValue(commandNames.INPUT_SOURCE, dataConvert.getBytes(StandardCharsets.UTF_8));
+						break;
+					case INPUT_TYPE:
+						isInputTypeDTV = LgLCDConstants.DTV.equalsIgnoreCase(value) ? true : false;
+						break;
+					case PMD_MODE:
+						dataConvert = LgLCDConstants.BYTE_COMMAND + EnumTypeHandler.getNameEnumByValue(PMDModeEnum.class, value);
+						sendRequestToControlValue(commandNames.PMD_MODE, dataConvert.getBytes(StandardCharsets.UTF_8));
+						break;
+					case PMD:
+						dataConvert = EnumTypeHandler.getNameEnumByValue(PowerManagement.class, value);
+						sendRequestToControlValue(commandNames.PMD, dataConvert.getBytes(StandardCharsets.UTF_8));
+						break;
+					case FAILOVER_MODE:
+						String inputPriority = group + LgLCDConstants.INPUT_PRIORITY;
+						String priorityInput = group + LgLCDConstants.PRIORITY_INPUT;
+						String priorityInputDown = group + LgLCDConstants.PRIORITY_DOWN;
+						String priorityInputUp = group + LgLCDConstants.PRIORITY_UP;
+						int failOverStatus = Integer.parseInt(value);
+						if (failOverStatus == LgLCDConstants.ZERO) {
+							sendRequestToControlValue(commandNames.FAILOVER, FailOverEnum.OFF.getValue().getBytes(StandardCharsets.UTF_8));
+							//Remove all priority 0,1,2,3.etc, priorityInput, and inputPriority.
+							stats.remove(inputPriority);
+							advancedControllableProperties.removeIf(item -> item.getName().equals(inputPriority));
 
-		//StaticStatistics
-		Map<String, String> statistics = new HashMap<String, String>();
+							stats.remove(priorityInput);
```

Comment:
we can provide the function to remove the controllable properties to avoid the reduplicate code.

For example
/**
	 * This method is used to remove unused statistics/AdvancedControllableProperty from {@link GudePDU8045Communicator#localExtendedStatistics}
	 *
	 * @param stats Map of statistics that contains statistics to be removed
	 * @param controls Set of controls that contains AdvancedControllableProperty to be removed
	 * @param listKeys list key of statistics to be removed
	 */
	private void removeUnusedStatsAndControls(Map<String, String> stats, List<AdvancedControllableProperty> controls, Set<String> listKeys) {
		for (String key : listKeys) {
			stats.remove(key);
			controls.removeIf(advancedControllableProperty -> advancedControllableProperty.getName().equals(key));
		}
	}